bogota-abril-2019/css-magazine-cover/Jefferson Alvarez Alvarez/portada/index.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Clara</title>
+        <meta charset="UTF-8">
+        <meta name="theme-color" content="#ffd63c">
+        <meta name="description" content="Mónica Cruz no envejece">
+        <meta name="keywords" content="Clara,Monica,Cruz,Edad">
+        <meta name="author" content="Jefferson Alvarez Alvarez">
+        <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Merriweather|Open+Sans:600,800|Roboto">
+        <link rel="stylesheet" type="text/css" href="./assets/css/style.css">
+        <link rel="icon" type="image/png" href="https://www.clara.es/favicon.ico">
+    </head>
+    <body>
+        <article class="main-container">
+            <section class="magazine-section">
+                <h1 class="title">CLARA</h1>
+            </section>
+            <section class="magazine-section">
+                <aside class="block-50">
+                    <section class="text-section">
+                        <p class="paragraph-large">¡FELIZ</p>
+                        <p class="paragraph-large">PELO</p>
+                        <p class="paragraph-large serif">NUEVO!</p>
+                        <p class="paragraph-medium">CORTES, COLORES</p>
```

Comment:
Seems info in the same context therefore should be under the same tag. Review Semantic

Reply:
I resolved it by replacing <p> by <span> and the general tag <section> of the paragraph by <p>